docs/spec/qualifiers.rst

```diff
@@ -272,33 +283,54 @@ details of the syntax:
     SmallInt = Annotated[int, ValueRange(0, 100)]
     SmallInt(1)  # Type error
 
+:pep:`593` and an earlier version of this specification used the term
+"annotations" instead of "metadata" for the extra arguments to
+``Annotated``. The term "annotations" is deprecated to avoid confusion
+with the parameter, return, and variable annotations that are part of
+the Python syntax.
+
+Meaning
+^^^^^^^
+
+The metadata provided by ``Annotated`` can be used for either static
+or runtime analysis. If a library (or tool) encounters an instance of
+``Annotated[T, x]`` and has no special logic for metadata element ``x``, it
+should ignore it and treat the expression as equivalent to ``T``. Thus, in general,
+any :term:`type expression` or :term:`annotation expression` may be
+wrapped in ``Annotated`` without changing the meaning of the
+wrapped expression. However, type
+checkers may additionally choose to recognize particular metadata elements and use
+them to implement extensions to the standard type system.
+
+``Annotated`` metadata may apply either to the base expression or to the symbol
+being annotated, or even to some other aspect of the program.
 
-Consuming annotations
-^^^^^^^^^^^^^^^^^^^^^
+Consuming metadata
+^^^^^^^^^^^^^^^^^^
 
-Ultimately, the responsibility of how to interpret the annotations (if
+Ultimately, deciding how to interpret the metadata (if
 at all) is the responsibility of the tool or library encountering the
 ``Annotated`` type. A tool or library encountering an ``Annotated`` type
-can scan through the annotations to determine if they are of interest
+can scan through the metadata to determine if they are of interest
 (e.g., using ``isinstance()``).
 
-**Unknown annotations:** When a tool or a library does not support
-annotations or encounters an unknown annotation it should just ignore it
-and treat annotated type as the underlying type. For example, when encountering
-an annotation that is not an instance of ``struct2.ctype`` to the annotations
-for name (e.g., ``Annotated[str, 'foo', struct2.ctype("<10s")]``), the unpack
-method should ignore it.
+**Unknown metadata:** When a tool or a library does not support
+metadata or encounters an unknown metadata element, it should ignore it
+and treat the annotation as the base expression. For example, when encountering
+a metadata object that is not an instance of ``struct2.ctype`` in the metadata
```

Comment:
I find this example (including its wording) really confusing, but that's probably because it requires knowledge of `struct2`. Since this was in the previous version of the spec, it's probably fine to leave it, but maybe we can clean it up in a subsequent round of changes.

Reply:
I'll just delete the example; I feel the text of the section is clear enough without it.